src/sentry/runner/__init__.py

```diff
@@ -46,19 +46,20 @@ def cli(ctx, config):
 
 # TODO(mattrobenolt): Autodiscover commands?
 map(lambda cmd: cli.add_command(import_string(cmd)), (
+    'sentry.runner.commands.backup.export',
```

Comment:
These _were_ sorted based on the command name, not the module path. I don't care so much, just stating why it was sorted how it was. :)